### PR TITLE
SOC-461 | Notifications - add notification to all users following

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-461

### CODE REVIEW

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+
+public class NotifyIntegrationTest extends AbstractIntegrationTest {
```
Comment (sqreek) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java]: please add @Category(IntegrationTest.class) for all integration tests

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+  }
+
+  private List<Long> addFollowers(FollowableItem item) {
+    List<Long> userIds = new ArrayList<>();
+    Long userId;
+    UriBuilder requestUrl;
+
+    for (int i = 0; i < 10; i++) {
+      userId = new Date().getTime();
```
Comment (sqreek) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java]: I'm still worry about using ``new Date().getTime()`` in tests.
especially in any kind of loop. Can You consider use ``getTime()`` once, and then, add ``i`` to it?

**Comment by sqreek** [comment]
except two comments, rest LGTM

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java (hunk 1 above)
Comment (kvas-damian) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java]: It's here: https://github.com/Wikia/pandora/blob/master/service/notifications/src/test/java/com/wikia/notificationsservice/integration/AbstractIntegrationTest.java#L22

File: service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java (hunk 2 above)
Comment (kvas-damian) [inline service/notifications/src/test/java/com/wikia/notificationsservice/integration/NotifyIntegrationTest.java]: We know about it and we're going to change that at some point. 
For now it does not generate any issues that's why I suggest keep as it is for some time.

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**